{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -12,22 +12,18 @@ license = "{{cookiecutter.license}}"
 readme = "README.md"
 include = ["README.md", "src/{{cookiecutter.__project_slug}}/schema", "project"]
 
-requires-python = ">=3.9"
+requires-python = ">=3.9,<4.0"
 
 dynamic = ["version"]
 
 dependencies = [
-  "linkml-runtime >=1.8.0",
+  "linkml-runtime >=1.1.24",
```

Comment:
Is it really necessary to support such an ancient version of linkml runtime?

Reply:
Probably not. I just kept what was there.